charts/incubator/notifiarr/questions.yaml

```diff
@@ -0,0 +1,113 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+                                - variable: env
+                                  group: "App Configuration"
+                                  label: "Image Environment"
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: DN_API_KEY
+                                        label: "DN_API_KEY"
+                                        description: "Notifiarr All Integrations API key from https://notifiarr.com"
+                                        schema:
+                                          type: string
+                                          default: "CHANGEME"
+                                      - variable: DN_UPSTREAMS_0
+                                        label: "DN_UPSTREAMS_0"
```

Comment:
Is this a list that you can add multiple? Is it comma separated?
maybe make it a type `list` and use helm to combine them.

Reply:
Yeah you can add multiple, need to be SPACE separated in the value field. Only one env var for DN_UPSTREAMS_0 can't do DN_UPSTREAMS_1 for example.

Reply:
then `join` can be used to aggregate a list into a single value

Probably will need to create a top level key like (in values.yaml):
```yaml
notifiarr:
  upstreams:
    - upstream1
    - upstream
    
    
then on env 
       .....
       env:
         DN_UPSTREAMS_0: "{{ join " " .Values.notifiarr.upstreams }}"
```

Same for DN_UI_PASSWORD

You'll need to adjust questions.yaml to reflect the above top level key structure.

Reply:
Just pushed a commit which I'm 90% sure does this. Great idea btw Stavros :).